src/components/Button/Button.style.ts

```diff
@@ -308,6 +316,11 @@ function buttonVariants({
         }
       `;
     case 'minimalTransparent':
+      const minimalTransparentActiveCSS = css`
```

Comment:
This style should be shared and added to an active psuedoclass

Reply:
done